cirq-core/cirq/vis/heatmap_test.py

```diff
@@ -218,7 +218,7 @@ def __format__(self, format_string):
 
     _, mesh = random_heatmap.plot(ax)
 
-    colormap = mpl.cm.get_cmap(colormap_name)
+    colormap = mpl.colormaps.get_cmap(colormap_name)
```

Comment:
Optional: Issue https://github.com/matplotlib/matplotlib/issues/20853 suggests we should perhaps just say

```python
colormap = mpl.colormaps[colormap_name]
```

Reply:
Done.  Maybe this will work in python 3.7 too.